stellar/deploy-contract.js

```diff
@@ -53,27 +53,25 @@ async function getInitializeArgs(config, chain, contractName, wallet, options) {
     }
 }
 
-async function processDeploy(options, config, chain) {
+async function deploy(options, config, chain) {
```

Comment:
```suggestion
async function deploy(contractName, options, config, chain) {
```
see comment in cli parsing

Reply:
updated.